source/reference/minio-mc/mc-support-diagnostics.rst

```diff
@@ -31,7 +29,7 @@ Requests that exceed this limit on a given host return a ``503`` error after ~10
 Use the :mc-cmd:`mc support diagnostics` command to trigger the diagnostic test.
 For clusters registered with SUBNET, the command uploads the results as part of SUBNET Health reports. 
 
-For airgapped or firewalled environments, or other environments that prevent direct network access from the deployment, you can save the report locally with the :mc-cmd:`~mc support diag --airgap` flag.
+For airgapped or firewalled environments, or other environments that prevent direct network access from the deployment, you can save the report locally with the :mc-cmd:`~mc support diagnostics --airgap` flag.
```

Comment:
```
 mc support -h
NAME:
  mc support - support related commands

USAGE:
  mc support COMMAND [COMMAND FLAGS | -h] [ARGUMENTS...]

COMMANDS:
  register           register with MinIO subscription network
  diag, diagnostics  upload health data for diagnostics

```

both flags seem to be valid. (`diag | diagnostics`)

Reply:
I'll need to add the short forms for flags in a later pass.